lambdas/models/cloudwatch_logs_query.py

```diff
@@ -0,0 +1,53 @@
+from dataclasses import dataclass
```

Comment:
I wonder if this belongs in the utils folder alongside the dynamo filters, as this is not really a models file?

Reply:
Thanks! Agree that this makes more sense to be in the utils folder. I will move them now.